target/linux/brcm63xx/base-files/etc/board.d/02_network

```diff
@@ -98,6 +98,11 @@ vr-3026e)
 		"0:lan:1" "1:lan:2" "2:lan:3" "3:lan:4" "8t@eth0"
 	;;
 
+vr-3022eu)
+	ucidef_add_switch "switch0" \
+		"0:lan:1" "1:lan:2" "2:lan:3" "3:lan:4" "5:lan:17" "8t@eth0"
```

Comment:
I'm not quite sure what the port index is good for. Apparently here I set it equal to the phy_id of the PHY connected to the respective Roboswitch port which is probably completely bogus.

Reply:
You labeled the port as "wan", so it should be "5:wan" here as well.

Reply:
I labeled it as "wan" in the board params as that's how it's labeled on the router itself. IIUC labeling it "wan" here assigns it the wan _role_ which may or may not be desired.

My thinking was that given it's the only 1Gb port of the router one may want to use it for the presumably faster internal network while using one of the FastEthernet ports as the "wan" port. If this doesn't sound right I'm happy to change it.

Reply:
I try to go by the principle of least surprise; if the stock firmware assigns this port as a wan port (and is labeled as such), then OpenWrt should do the same by default.

Reply:
Fair enough.